examples/mysql/package.json

```diff
@@ -28,14 +28,14 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.18.0",
-    "@opentelemetry/exporter-jaeger": "^0.18.0",
-    "@opentelemetry/exporter-zipkin": "^0.18.0",
-    "@opentelemetry/instrumentation": "^0.18.0",
-    "@opentelemetry/node": "^0.18.0",
-    "@opentelemetry/plugin-http": "^0.18.0",
-    "@opentelemetry/plugin-mysql": "^0.14.0",
-    "@opentelemetry/tracing": "^0.18.0",
+    "@opentelemetry/api": "^0.18.1",
+    "@opentelemetry/exporter-jaeger": "^0.18.2",
+    "@opentelemetry/exporter-zipkin": "^0.18.2",
+    "@opentelemetry/instrumentation": "^0.18.2",
+    "@opentelemetry/instrumentation-http": "^0.18.2",
```

Comment:
I think moving from plugin-http to instrumentation-http requires to add `registerinitailization()` in sample code.